src/test/ValidateRedeemFei.sol

```diff
@@ -23,24 +37,31 @@ contract ValidationRedeemFei is Test {
 
     address public constant A_FEI = 0x683923dB55Fead99A79Fa01A27EeC3cB19679cC3;
 
-    address public constant DAI = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address public constant A_DAI = 0x028171bCA77440897B824Ca71D1c56caC55b68A3;
 
     address public constant AAVE_MAINNET_RESERVE_FACTOR = 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c;
+    
+    address public constant AAVE_LENDING_POOL = 0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9;
+
+    IFixedPricePSM public constant DAI_FIXED_PRICE_PSM = IFixedPricePSM(0x2A188F9EB761F70ECEa083bA6c2A40145078dfc2);
 
     // can't be constant for some reason
     string internal MARKET_NAME = "AaveV2Ethereum";
 
-    RedeemFei public feiPayload;
+    RedeemFei feiPayload;
+    AFeiToDaiSwapper swapper;
 
     function setUp() public {
-        feiPayload = new RedeemFei();
+        swapper = new AFeiToDaiSwapper();
+        feiPayload = new RedeemFei(address(swapper));
     }
 
     function testProposalPostPayload() public {
 
-        uint256 aFeiAmount = 300_000e18;
-        uint256 daiBalanceBefore = IERC20(DAI).balanceOf(AAVE_MAINNET_RESERVE_FACTOR);
-        uint256 aFeiBalanceBefore = IERC20(A_FEI).balanceOf(AAVE_MAINNET_RESERVE_FACTOR);
```

Comment:
Can we have the check for aFeiBalanceBefore and aFeiBalanceAfter as well to show that it's all been used up?

Reply:
It shouldnt be all used up until the last time the function is called, maybe I'm misunderstanding

Reply:
Oh I meant to just have a check for the aFeiBalanceBefore and aFeiBalanceAfter as well. You should be able to calculate that in the test right?